package.json

```diff
@@ -112,7 +112,7 @@
     },
     {
       "path": "./packages/react/dist/browser/runtime.js",
-      "limit": "445B",
+      "limit": "729B",
```

Comment:
I know :(
However if the changes proposed in this PR are considered the way to go then perhaps there are ways to reduce this.